engine/taskpool-collector/src/main/kotlin/io/holunda/camunda/taskpool/sender/gateway/AxonCommandListGateway.kt

```diff
@@ -44,3 +47,26 @@ open class AxonCommandListGateway(
   }
 
 }
+
+/**
+ * Error handler, logging the error.
+ */
+open class LoggingTaskCommandErrorHandler(private val logger: Logger) : TaskCommandErrorHandler {
+
+  override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+    logger.error("SENDER-006: Sending command $commandMessage resulted in error ${commandResultMessage.exceptionResult()}")
+  }
+}
+
+/**
+ * Logs success.
+ */
+open class LoggingTaskCommandSuccessHandler(private val logger: Logger) : TaskCommandSuccessHandler {
+
+  override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+    logger.debug("SENDER-004: Successfully submitted command $commandMessage")
```

Comment:
if `logger` is a standard slf4j logger and not a kotlin-logging one, we should either surround the call with `if(logger.isDebugEnabled())` or use the slf4j placeholder syntax to avoid building strings that will never be logged.